internal/cephfs/volume.go

```diff
@@ -44,23 +44,12 @@ const (
 	modeAllRWX = 0777
 )
 
-// Subvolume holds subvolume information.
+// Subvolume holds subvolume information. This includes only the needed members
+// from fsAdmin.SubVolumeInfo.
 type Subvolume struct {
-	BytesQuota    int      `json:"bytes_quota"`
-	DataPool      string   `json:"data_pool"`
-	Features      []string `json:"features"`
-	GID           int      `json:"gid"`
-	Mode          int      `json:"mode"`
-	MonAddrs      []string `json:"mon_addrs"`
-	Path          string   `json:"path"`
-	PoolNamespace string   `json:"pool_namespace"`
-	// The subvolume "state" is based on the current state of the subvolume.
-	// It contains one of the following values:
-	// * "complete": subvolume is ready for all operations.
-	// * "snapshot-retained": subvolume is removed but its snapshots are retained.
-	State string `json:"state"`
-	Type  string `json:"type"`
```

Comment:
@nixpanic just wondering is it a good idea to drop these fields. It helps in one way to fill it if needed and also gives us some view on possible options available in the response. I dont have any hard feeling on this, but somehow in a mindset to keep it even if we are not using it.

Reply:
I left the reference to [cephfs/admin.SubVolumeInfo](https://godoc.org/github.com/ceph/go-ceph/cephfs/admin#SubVolumeInfo) so that it is easy to check what other members are available. Not filling unused members is a little more efficient, cleaner and improves readability a bit.